WPMedia/fusion-news-theme-blocks
# TMEDIA-742 : Medium Promo block

## Description

Update Medium Promo Block to Themes v2 architecture.

## Jira Ticket

- [TMEDIA-742 : Medium Promo Block](https://arcpublishing.atlassian.net/browse/TMEDIA-742)

## Acceptance Criteria

- Medium Promo block is migrated to the new architecture.
- Dividing lines between items in the block are included, but any dividing lines framing the block will be done at the layout level

## Test Steps

### Blocks Repo : Storybook, Testing, Linting
1. Checkout this branch `git checkout TMEDIA-742-medium-promo`
2. Start Storybook with `npm run storybook`. Verify stories for Medium Promo.

### Feature Pack

1. Change local feature pack branch with `git checkout TMEDIA-742-medium-promo`
2. Ensure `THEMES_BLOCKS_REPO` in `.env` points to local blocks repo.
3. Start Fusion w/ `npx fusion start -f -l @wpmedia/medium-promo-block`
4. From local PB Editor:
     a. Create new page and add `Medium Promo – Arc Block`.
     b. Set "Display Content Info" in block to `ans-item`, `content-api`, `7TLCY4UZN5F4BA5I4FP62E6NYE`.
     c. Confirm block functionality and cosmetics.


Test with the follow stories
* Video Featured Media - website_url = `/2022/08/31/story-with-video-featured-media/`
* Gallery Featured Media - website_url = `/2022/08/31/story-with-gallery-featured-media/`
* No featured Media - website_url = `/2022/08/26/this-is-a-test-story/`


<img width="1626" alt="Microsoft Edge-2022-08-31-21-32 17" src="https://user-images.githubusercontent.com/868127/187776480-a51faeaf-42f0-4d87-8d23-d5a49eb8181f.png">



## Note
This topic branch was accidentally branched from another topic branch. Please disregard all commits in this topic branch prior to [d26ad32](https://github.com/WPMedia/arc-themes-blocks/pull/1481/commits/d26ad3249db2cab85ff8f06d385376431066f4ec). That all all subsequent commits represent work for this pull request.

## Testing

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
